WordPress/src/main/java/org/wordpress/android/ui/posts/editor/media/EditorMediaListener.kt

```diff
@@ -0,0 +1,12 @@
+package org.wordpress.android.ui.posts.editor.media
+
+import org.wordpress.android.fluxc.model.PostImmutableModel
+import org.wordpress.android.ui.posts.EditPostActivity.OnPostUpdatedFromUIListener
+import org.wordpress.android.util.helpers.MediaFile
+
+interface EditorMediaListener {
```

Comment:
No need to change anything. I'm just sharing my thoughts about this listener.

We utilized this listener in the EditPostActivity even though it was never a perfect solution. It can very easily lead to a huge memory leak when the VM keeps reference to the listener (activity) after the activity is destroyed. Ideally, we'd be utilizing livedata or some other form of communication between the view and the vm layers.